src/common/SurgePatch.cpp

```diff
@@ -632,6 +632,24 @@ SurgePatch::SurgePatch(SurgeStorage *storage)
             Surge::Formula::createInitFormula(fs);
         }
 
+    // Build table of param_ptr -- storage name
+    for (auto *p : param_ptr)
+        param_ptr_by_storagename[p->get_storage_name()] = p;
+
+    /* ---- Set any OSC names here  ---- */
```

Comment:
Are these comments supposed to be here? There's an example function call given but I'm not sure when we'd use this.

Reply:
I think that's the mechanism that will be used to splice in OSC names on an as-needed basis. I could be wrong about that, though.

Reply:
You're not wrong about that 🙂